src/libData/AccountData/AccountStoreSC.h

```diff
@@ -196,10 +198,10 @@ class AccountStoreSC : public AccountStoreBase<MAP> {
                          const boost::multiprecision::uint128_t& balance,
                          bool& ret, TransactionReceipt& receipt);
 
-  void InvokeEvmInterpreter(INVOKE_TYPE invoke_type, RunnerDetails& details,
-                            const uint32_t& version, bool& ret,
-                            TransactionReceipt& receipt,
-                            struct EvmReturn& result);
+  uint64_t InvokeEvmInterpreter(Account* account, INVOKE_TYPE invoke_type,
```

Comment:
is this the contract account or the caller account?   Might reflect in the name.

Reply:
For Evm it is the contact account, it has been renamed to reflect this, will see that on next commit